check-pro-mode.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs')
+
+let proMode = false
+
+try {
+  fs.accessSync('./.pro-mode')
```

Comment:
Could simply be `proMode = fs.existsSync('.pro-mode')`

Reply:
Good point. :)